src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    private final Random random = new Random();
+
+    public Colors getRandomColor() {
+        int index = random.nextInt(Colors.values().length);
+        return Colors.values()[index];
```

Comment:
```suggestion
        return Colors.values()[index].name();
```

Reply:
What do you mean adding .name()? I did not have such method